src/Cli/Utils.cs

```diff
@@ -259,14 +258,15 @@ public class LowerCaseNamingPolicy : JsonNamingPolicy
 
         /// <summary>
         /// Returns the Serialization option used to convert objects into JSON.
+        /// Not escaping any special unicode characters.
         /// Ignoring properties with null values.
         /// Keeping all the keys in lowercase.
         /// </summary>
         public static JsonSerializerOptions GetSerializationOptions()
         {
             JsonSerializerOptions? options = new()
             {
-                Encoder = JavaScriptEncoder.Create(UnicodeRanges.All),
+                Encoder = JavaScriptEncoder.UnsafeRelaxedJsonEscaping,
```

Comment:
The docs say: https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/character-encoding#serialize-all-characters to use this with caution. 
Can we use `SerializeToUtf8Bytes` so we can use this encoder safely?

Reply:
it creates a Byte array and then we would have convert it again to string. Furthermore, Under the caution, it is mentioned `Never allow the raw UnsafeRelaxedJsonEscaping output to be emitted into an HTML page or a <script> element`. Our usage doesn't violate this rule, instead it's directly consumed as a file. Secondly, this serialization option is in CLI only and used while generating the config. As CLi is meant for local development. this should be fine, I believe.

Reply:
we already use WriteAllText for generating the config: [here](https://github.com/Azure/data-api-builder/blob/ee22ea3a46f8b983ffe756a53b5e87bef022382b/src/Cli/Utils.cs#L877), which uses UTF-8 for default encoding as pointed out [here](https://github.com/Azure/data-api-builder/pull/1210/files#r1139458200) in the discussion.